wurst/_wurst/Orders.wurst

```diff
@@ -0,0 +1,371 @@
+	static constant neutralinteract			 = 852566
+	static constant neutralspell			 = 852630
+	static constant nightelfbuild			 = 851997
+	static constant orcbuild			 = 851996
+	static constant parasite			 = 852601
+	static constant parasiteoff			 = 852603
+	static constant parasiteon			 = 852602
+	static constant patrol				 = 851990
+	static constant phaseshift			 = 852514
+	static constant phaseshiftinstant		 = 852517
+	static constant phaseshiftoff			 = 852516
+	static constant phaseshifton			 = 852515
+	static constant phoenixfire			 = 852481
+	static constant phoenixmorph			 = 852482
+	static constant poisonarrows			 = 852255
+	static constant poisonarrowstarg		 = 852254
+	static constant polymorph			 = 852074
+	static constant possession			 = 852196
+	static constant preservation			 = 852568
+	static constant purge				 = 852111
+	static constant rainofchaos			 = 852237
+	static constant rainoffire			 = 852238
+	static constant raisedead			 = 852197
+	static constant raisedeadoff			 = 852199
+	static constant raisedeadon			 = 852198
+	static constant ravenform			 = 852155
+	static constant recharge			 = 852157
+	static constant rechargeoff			 = 852159
+	static constant rechargeon			 = 852158
+	static constant rejuvination			 = 852160
+	static constant renew				 = 852161
+	static constant renewoff			 = 852163
+	static constant renewon				 = 852162
+	static constant repair				 = 852024
+	static constant repairoff			 = 852026
+	static constant repairon			 = 852025
+	static constant replenish			 = 852542
+	static constant replenishlife			 = 852545
+	static constant replenishlifeoff		 = 852547
+	static constant replenishlifeon			 = 852546
+	static constant replenishmana			 = 852548
+	static constant replenishmanaoff		 = 852550
+	static constant replenishmanaon			 = 852549
+	static constant replenishoff			 = 852544
+	static constant replenishon			 = 852543
+	static constant request_hero			 = 852239
+	static constant requestsacrifice		 = 852201
+	static constant restoration			 = 852202
+	static constant restorationoff			 = 852204
+	static constant restorationon			 = 852203
+	static constant resumebuild			 = 851999
+	static constant resumeharvesting		 = 852017
+	static constant resurrection			 = 852094
+	static constant returnresources			 = 852020
+	static constant revenge				 = 852241
+	static constant revive				 = 852039
+	static constant roar				 = 852164
+	static constant robogoblin			 = 852656
+	static constant root				 = 852165
+	static constant sacrifice			 = 852205
+	static constant sanctuary			 = 852569
+	static constant scout				 = 852181
+	static constant selfdestruct			 = 852040
+	static constant selfdestructoff			 = 852042
+	static constant selfdestructon			 = 852041
+	static constant sentinel			 = 852182
+	static constant setrally			 = 851980
+	static constant shadowsight			 = 852570
+	static constant shadowstrike			 = 852527
+	static constant shockwave			 = 852125
+	static constant silence				 = 852592
+	static constant sleep				 = 852227
```

Comment:
where is `stun` ?

Reply:
whoa, I didn't know about it. Anyway, I've taken the order list from cjass, and it can miss much of them, so I assume it can be added later, or if I will be given an id of that order.

Reply:
```
Hidden Order Ids:
851971 (smart): This is a point or object targeted order that is like a right click.
852000 (skillmenu): This is an order with no target that opens the skill menu of heroes. If it is issued for a normal unit with triggers it will black out the command card for this unit, the command card will revert to normal after reselecting the unit.
851994 (buildmenu): This is an order with no target that opens up the build menu of a unit that can build structures.
852002 to 852007 (moveslot): These are item targeted orders that move the target item to a certain inventory slot of the ordered hero. The id 852002 will move it to slot 1, the id 852003 will move it to slot 2 and so on.
852008 to 852013 (useslot): These are orders that will make the ordered hero use the item in a certain inventory slot. If it's an order with no target or object or point targeted depends on the type of item. The id 852008 will use the item in slot 1, the id 852009 will use the item in slot 2 and so on.
851976 (cancel): This is an order with no target that is like a click on a cancel button. We used to be able to catch cancel clicks with this id back then but this id doesn't seem to work any more.
851973 (stunned): This order is issued to units that get stunned by a spell, for example War Stomp (AOws). This is probably a hold position + hold fire order. The ordered unit will be unable to move and attack.
```

From: http://www.wc3c.net/showthread.php?t=81742